ferrilata-devilline/Controllers/ViewControllers/ViewBadgesController.cs

```diff
@@ -20,8 +18,7 @@ public ViewBadgesController(IBadgeService badgeService)
         [HttpGet("/badgelibrary")]
         public IActionResult GetBadgeLibrary()
         {
-            List<Badge> badges = _badgeService.GetAll();
-            return View(badges);
+            return View(_badgeService.GetAllDTO());
```

Comment:
It is much better if one line of code is responsible for one action. Here you are returning a view and calling a method. Now imagine if a bug comes up in the future. It's impossible to debug the code like this. Please first save the result of _bagservice.GetAllDTO() to a variable and after that pass it to the view.

Reply:
Done